doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -307,21 +307,16 @@ You can schedule actors or tasks to a placement group using
 
 .. note::
 
-  When you use an actor with a placement group, always specify ``num_cpus``.
+  By default, Ray actors use 1 logical CPU for scheduling, and 0 logical CPU for running.
+  This means, by default, actors cannot get scheduled on a zero-cpu node, but an infinite number of them can run on any non-zero cpu node.
+  The default resource requirements for actors was chosen for historical reasons.
+  When scheduling an actor with default resource requirements and a placement group, the placement group has to be created with a bundle containing at least 1 CPU
+  (since the actor requires 1 CPU for scheduling). However, after the actor is created, it doesn't consume any placement group resources.
```

Comment:
```suggestion
  By default, Ray actors require 1 logical CPU to be available at schedule time, but after being scheduled, they do not acquire any CPU resources.
  In other words, by default, actors cannot get scheduled on a zero-cpu node, but an infinite number of them can run on any non-zero cpu node.

  Thus, when scheduling an actor with the default resource requirements and a placement group, the placement group has to be created with a bundle containing at least 1 CPU
  (since the actor requires 1 CPU for scheduling). However, after the actor is created, it doesn't consume any placement group resources.
```

Does the 1 CPU need to be *available* or can the actor still be scheduled even if there is another task/actor using the CPU? If it is the latter, probably need to update my suggestion.

Reply:
Yea, needs to be available.

Reply:
Needs to be available.